build.gradle

```diff
@@ -81,6 +82,14 @@ tasks.getByName("runIde").configure {
     }
 }
 
+task createProperty(type: WriteProperties) {
```

Comment:
Not sure I understand what this means. If I build with a SENTRY_DSN env variable than it will report to sentry, otherwise not?

I think we need to make sure that production builds always have sentry, while other not, at least by default.

And for non production builds, maybe report to a different project or at least tag these so we know they are not production errors